packages/web-components/src/calendar/calendar.styles.ts

```diff
@@ -148,14 +151,59 @@ ${display("inline-block")} :host {
 `.withBehaviors(
   forcedColorsStylesheetBehavior(
       css`
-          .day.selected {
-              color: ${SystemColors.Highlight};
-          }
-
-          .today .date {
-              background: ${SystemColors.Highlight};
-              color: ${SystemColors.HighlightText};
-          }
+        :host {
+          forced-color-adjust: auto;
+          background: ${SystemColors.ButtonFace};
+        }
+        .day,
+        .interact .day,
+        .week-day {
+            background: ${SystemColors.Canvas};
+            color: ${SystemColors.CanvasText};
+            fill: currentcolor;
+            border-color: ${SystemColors.Canvas}
+        }
+        .interact .day:not(.disabled, .inactive):hover {
+            background: ${SystemColors.ButtonFace};
+            color: ${SystemColors.ButtonText};
+            outline: 1px solid ${SystemColors.Highlight};
+        }
+        .week-day:${focusVisible},
+        .interact .day:not(.disabled, .inactive):${focusVisible} {
+            forced-color-adjust: none;
+            background: ${SystemColors.ButtonFace};
+            border-color: transparent;
+            ${focusTreatmentBase}
```

Comment:
If an element is able to be focused, this helper should be applied on the regular styles only. For HC, only set `outline-color` to the necessary `SystemColors` value.

Reply:
created a focus visible selector in regular styles and removed the helper here.